services/account_service_test.go

```diff
@@ -161,3 +141,61 @@ func TestAccountBalance_Online_Historical(t *testing.T) {
 
 	mockIndexer.AssertExpectations(t)
 }
+
+func TestAccountCoins_Online(t *testing.T) {
```

Comment:
Accounts are deprecated in zend, all accounts are hard coded to empty string `""` in RPC commands.